### ISSUE TITLE
New Module: Pill

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/skin#69]
More details and spec to follow. @JeffNixonCreative can you please add?

This may end up being part of the button module, as to me it seems like just a rounded version of buttons and fake buttons.

<img width="899" alt="screen shot 2018-02-26 at 10 31 10 am" src="https://user-images.githubusercontent.com/38065/36688077-383fa292-1ae0-11e8-8b25-bc886c0473fa.png">

Latest designs:
https://ebay.invisionapp.com/share/CWN9X1JF6MA#/screens/293357159

**Comment by ianmcburnie** [comment]
Waiting on further clarification on multi-select use case, and multiline use case before proceeding.

**Comment by JeffNixonCreative** [comment]
[Pills_SKIN_v1.pdf](https://github.com/eBay/skin/files/1812945/Pills_SKIN_v1.pdf)

**Comment by msendlakowski** [comment]
Latest designs:
https://ebay.invisionapp.com/share/CWN9X1JF6MA#/screens/293357159

I think Ian's point about making it a special class of button makes sense, unless we find something in the design docs that points to creating it as a different component.

**Comment by ianmcburnie** [comment]
Feature merged into 7.0.0 branch.
